slides/src/content/a11y.md

```diff
@@ -0,0 +1,142 @@
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Design
+
+Designers can include details of the following accessibility and design considerations to overcome challenges previously mentioned:
+
+- Format: Use a standard page template, ensuring content is read like a book - left to right, top to bottom
+- Fonts: Use sans serif fonts and set a base pixel measure at 1em = 16px (=12pt)
+- Colour:  Ensure foreground:background ratios are 4:5:1 for normal text and 3:1 for large text
+- Images: Images must be alt tagged or have figure numbers describing the image
+- Tables:  Do not use tables for layout, only data, and explicitly label columns and rows
+- Links and Buttons: Must be descriptive (e.g. 'complete order' instead of 'click here'), must indicate the action of the link or button ('this will open in a new window')
+- Modals should be developed for information purposes and not have complex interactive elements.  Modals should limit content if possible for simplicity 
+- Pages with lengthy content should have clear markers indicating the scope of the content and provide the ability to jump to key points including the beginning and end
+
+
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Development
+
+To acheive accessibility set out in design, developers can follow these do's and don'ts
+
+- Attempt to pin down client requirements for browsers to be supported
+- Ensure proper WAI-ARIA tagging within our JS frameworks (use of `aria-describedby` for example) to ensure components and widgets on screen are accessible
+- Use the Title attribute on HTML anchor links to add more information about a link and/or its consequences
+- Use the non-standard 'datatable=0' attribute on the Table tags to tell JAWS to ignore layout tables
+- Don't use the 'summary' attribute on the Table tag if you must create a layout table (or set the 'summary' attribute to null
+- Don't use TH or Caption tags on layout tables
+- Use relative width and font sizing over absolutes (this may be obvious but...)
+- Use headers in sequnce (H1 - H6)
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Development (Continued)
+
+- To help navigation for visually impaired users, consider developing an offscreen-text CSS class to further mark up headers and indicate start and end of sections
+
+```ts
+	.offscreen-text {
+     	left: -3000px;
+     	position: absolute;
+     	width: 500px;
+     	font-size: 100%;}
+```
+- Mark up text with this class beside sections, including those ARIA tagged with `<aside>`, `<main>`, `<nav>`, `<footer>`, `<header>` to indicate where the user is
+- Examples:
+  - 'start of navigation'
+  - 'end of navigation'
+
+
+---
+<!-- .slide: id="a11y" -->
+## Testing Accessibility 
+
+Testing accessibility involves testing the combination of design considerations and how they are implemented in code.
+
+- Accessibility testing should typically start with manual testing then automated
+  - Develop appropriate scripts
+  - Automate using any number of tools
```

Comment:
These three points are true of all kinds of testing - we should skip anything that's generic.